www/install/php/Update-21.04.0-beta.1.php

```diff
@@ -186,19 +186,68 @@
         "SELECT config_id FROM cfg_centreonbroker"
     );
     while ($row = $stmt->fetch(\PDO::FETCH_ASSOC)) {
-        $pearDB->query(
+        $brokerLogData = [
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 1,
+                'id_level' => 5
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 2,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 3,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 4,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 5,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 6,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 7,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 8,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 9,
+                'id_level' => 3
+            ],
+            [
+                'id_centreonbroker' => $row['config_id'],
+                'id_log' => 10,
+                'id_level' => 3
+            ]
+        ];
+        $statement = $pearDB->prepare(
             "INSERT INTO `cfg_centreonbroker_log` (`id_centreonbroker`, `id_log`, `id_level`)
-            VALUES (" . $row['config_id'] . ",1,5),
-                   (" . $row['config_id'] . ",2,3),
-                   (" . $row['config_id'] . ",3,3),
-                   (" . $row['config_id'] . ",4,3),
-                   (" . $row['config_id'] . ",5,3),
-                   (" . $row['config_id'] . ",6,3),
-                   (" . $row['config_id'] . ",7,3),
-                   (" . $row['config_id'] . ",8,3),
-                   (" . $row['config_id'] . ",9,3),
-                   (" . $row['config_id'] . ",10,3)"
+            VALUES (:id_centreonbroker, :id_log, :id_level)"
```

Comment:
Why did you replace a database query with 10?

Reply:
@callapa Do you mean why i created the array ? i didn't not understand can you explain more please ? ^^ thanks